Cargo.toml

```diff
@@ -31,13 +33,18 @@ alloy-rlp = { version = "0.3.8", features = ["derive"] }
 alloy-primitives = { version = "0.7.7" }
 
 # near
-near-primitives = { version = "0.24.1" }
-near-crypto = { version = "0.24.1" }
+near-primitives = { version = "0.25.0" }
+near-crypto = { version = "0.25.0" }
+near-workspaces = { version = "0.12.0", features = [
+    "experimental",
+    "unstable",
+] }
+near-jsonrpc-client = { git = "https://github.com/edsonalcala/near-jsonrpc-client-rs.git", branch = "master" }
```

Comment:
let's get a fixed/tag version? otherwise this going to break for someone

Reply:
yes, fixed